components/NewPersonView/Layout.tsx

```diff
@@ -103,6 +105,20 @@ const Layout = ({ person, children }: Props): React.ReactElement => {
       href: `/people/${person.id}/case-note`,
     },
   ];
+
+  if (
+    isFeatureActive('case-status') &&
+    casestatus &&
+    casestatus.caseStatuses &&
+    !groupCaseStatusByType(casestatus.caseStatuses).has('CIN') &&
```

Comment:
Is `!groupCaseStatusByType(casestatus.caseStatuses).has('CIN')`, checking that the case statuses retrieved do not include `CIN`?

Reply:
The check disables the "Add a case status" if there currently are active CIN for the user.
The main reason for this is because you can't have more than one CIN, the form would return an error in case you try to add another one.

It needs to be removed/adjusted when we add other case status types to the flow